app/dashboard/models.py

```diff
@@ -816,6 +816,8 @@ class Interest(models.Model):
 
     profile = models.ForeignKey('dashboard.Profile', related_name='interested', on_delete=models.CASCADE)
     created = models.DateTimeField(auto_now_add=True, blank=True, null=True)
+    has_question = models.BooleanField(default=False)
+    issue_message = models.TextField(default='', blank=True)
```

Comment:
you need to create a migraiton for these model fields via `./manage.py makemigraitons`

Reply:
and commit it back up

Reply:
32f56bf8ecc4c3dc5bb83dee40becb9d5ad40511 adds these migrations